pkgs/by-name/no/notion-app/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+  testers,
+  libplist,
+  jq,
+  writeText,
+  runCommand,
+}:
+let
+  info =
+    (builtins.fromJSON (builtins.readFile ./info.json))."${stdenvNoCC.targetPlatform.system}"
+      or (throw "notion-app: unsupported system ${stdenvNoCC.targetPlatform.system}");
+in
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "notion-app";
+  version = info.version;
+  src = fetchurl { inherit (info) url hash; };
+  sourceRoot = ".";
+  nativeBuildInputs = [ unzip ];
+  phases = [
+    "unpackPhase"
+    "installPhase"
+  ];
```

Comment:
Don't ever override `phases`. Instead set `dontConfigure`/`dontBuild`/... to true.